WooCommerce/WooCommerceTests/ViewRelated/Products/BottomSheetListSelector/ProductsSortOrderBottomSheetListSelectorCommandTests.swift

```diff
@@ -4,16 +4,25 @@ import XCTest
 
 final class ProductsSortOrderBottomSheetListSelectorCommandTests: XCTestCase {
     func testInitialSelectedValue() {
```

Comment:
We can rename this to clarify it further:

```suggestion
    func test_initial_selected_value_is_stored_in_command() {
```

This is using a very old test method convention. Our new conventions use snake case: https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/naming-conventions.md#test-methods. 

Also, the `testInitialSelectedValue` does not really explain what is _expected_. My suggestion isn't that great but it's probably at least a little bit better than what we have here. 🤪 

I don't think this test is very helpful but it doesn't harm anyone to keep it.